bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Relation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Runtime.Serialization;
```

Comment:
Move

Reply:
Will do tomorrow. Didn't pay attention when it added the using. Not gonna answer every one of the "Move" conversations. Once one is done I just resolve.